packages/core/database/lib/query/query-builder.js

```diff
@@ -77,6 +78,14 @@ const createQueryBuilder = (uid, db) => {
       return this;
     },
 
+    max(column) {
+      state.type = 'max';
+      state.max = column;
```

Comment:
shouldn't we put that in the state.select so it actually uses the logic of prefixing etc that is already implemented ?

Reply:
I'm not sure to understand.
How do you use select with `qb.max(helpers.toColumnName(meta, state.max), { as: state.max });` ?

Reply:
I mean instead of doing state.max = col you could actually use state.select = col
 as it is in reality a normal select behind the hood and should follow the same rules around prefixes toCOlumnName and such
 
 And do qb.max(state.select)